tests/phpunit/integration/Core/Consent_Mode/REST_Consent_Mode_ControllerTest.php

```diff
@@ -323,92 +323,92 @@ public function test_get_ads_measurement_status() {
 
 	public function test_get_ads_measurement_status__ads_module_connected() {
 		$this->setup_rest();
-		// Setup the site and admin user to make a successful REST request.
 		$this->grant_manage_options_permission();
 
-		$this->force_module_connection( Ads::MODULE_SLUG );
+		add_filter(
```

Comment:
More importantly than the other comments on this test, the test is intended to be specific to the Ads-module connected scenario, however the current implementation bypasses this and essentially forces a `true` result.

While the tests here should no longer be module specific (since the details of whether or not ads is connected are delegated to modules), we need to make sure that updates to tests make sense and if not, remove them or rework them more substantially.

The tests for `get_ads_measurement_status` should be focused on the workings of that endpoint alone, which should exercise its handler to make sure it's working as expected.